tests/e2e/e2e.sh

```diff
@@ -90,10 +46,18 @@ do
 	echo "---" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
```

Comment:
please delete this echo line - it is no longer needed as I added YAML terminators in a prior PR to all of the yaml files.  I am not sure if my master PR was backported - if not it needs to be.  I will look at this now and make it so.

Reply:
Fixed!